### PR TITLE
Fix bug on window state save

### LINKED ISSUE #433: GUI stays on second monitor even if it doesn't exist
The GUI saves the monitor it's on, but it should check if that monitor is connected and fall back to the main monitor if it doesn't exist.

- Example 1: GUI was on main monitor, main monitor is disconnected so second monitor becomes the main monitor: everything is fine :3
- Example 2: GUI was on second monitor, main monitor is disconnected so second monitor becomes main monitor: GUI window is invisible and inaccessible as it's on the second monitor, but it doesn't exist.

@loucass003

### PR DESCRIPTION
**Pull request by ImUrX**
Fixes #433

### CODE REVIEW

File: gui/src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
         "decorations": false,
         "transparent": false,
         "fileDropEnabled": false,
-        "visible": true
+        "visible": false
```
Comment (Louka3000) [inline gui/src-tauri/tauri.conf.json:81]:
```suggestion
        "visible": true
```
This needs to be true or else the window is invisible .-.

**Review by Louka3000 — CHANGES_REQUESTED** [review]
Doesn't work for me

File: gui/src-tauri/tauri.conf.json (hunk 1 above)
Comment (ImUrX) [reply gui/src-tauri/tauri.conf.json:81]: That’s not what visible means, and weird I tested it in my side

Comment (Louka3000) [reply gui/src-tauri/tauri.conf.json:81]: The window is invisible if false and visible if true tho…

Comment (Louka3000) [reply gui/src-tauri/tauri.conf.json:81]: Nvm, after further testing, ig that doesn't matter.
This still doesn't fix the window bug for me tho

**Review by Louka3000 — APPROVED** [review]
Works now OWO

**Review by Eirenliel — APPROVED** [review]

**Comment by Beanow** [comment]
Please note the PR you're referencing was closed: https://github.com/tauri-apps/tauri-plugin-window-state/pull/52#issuecomment-1369853832

> Hey :wave: Thank you for your contributions! We don't accept PRs for the individual plugin repos anymore since we're moving all of them to https://github.com/tauri-apps/plugins-workspace. It would be much appreciated if you could re-open it against that repo.

**Comment by ImUrX** [comment]
Yeah. I know, I got the notification XD

**Pull request merged**